Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -73,12 +74,25 @@ namespace ROS2::Utils
                     fullSourcePath.c_str(),
                     crc);
 
-                AvailableAsset t;
-                t.m_sourceAssetRelativePath = info.m_relativePath;
-                t.m_assetId = info.m_assetId;
-                t.m_sourceAssetGlobalPath = fullSourcePathStr;
-                t.m_productAssetRelativePath = info.m_relativePath;
-                availableAssets[crc] = t;
+                AvailableAsset foundAsset;
```

Comment:
question: This code looks very similar to that from another PR? Maybe once the other is merged this diff will go away? Just looks like this PR has some other unrelated changes in it potentially?

Reply:
Yes, it was rebased on top of PR that we already merged and squashed. I will rebase.

Reply:
Okay great thanks :+1: